CHANGELOG.md

```diff
@@ -63,6 +65,10 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Protobuf serialization for crypto types are now implemented in `exonum-crypto`
   crate (#1496).
 
+#### exonum-merkledb
+
+- Dot symbol is not allowed in index names and group prefixes anymore . (#1558)
```

Comment:
This looks like a bogus description. The dot symbol is not allowed in:

- `Prefixed` accesses
- index / group / component suffixes **in the `FromAccess` derive macro** (which is implemented in #1562; I'll update it accordingly)

Indexes, groups, etc. created manually should still allow dots in their names.